src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala

```diff
@@ -63,6 +63,7 @@ object MockWorkspaceServer {
     useCallCache = Option(randomBoolean()),
     deleteIntermediateOutputFiles = Option(randomBoolean()),
     useReferenceDisks = Option(randomBoolean()),
+    userComment = Option("This submission came from a mock server."),
```

Comment:
I am still trying to understand what sort of unit test coverage makes sense in the orchestration level. @salonishah11 , is this test change (and the one below) sufficient to cover the new end points?

Reply:
I think these 2 unit tests cover the addition of userComment in create submission API. We should add unit tests for the new PATCH userComment submission API.